jdatetime/__init__.py

```diff
@@ -577,10 +564,22 @@ def strftime(self, format):
         try:
             if self.hour > 12:
                 format = format.replace("%I", '%02.d' % (self.hour - 12))
-            else:
+            elif self.hour > 0:
```

Comment:
hmm it took me a while to understand this `if/else`.
What do you think if it's like

```python
if self.hour > 12:
    ...
elif self.hour == 12:
   ...
else:
   ...
```

Could you add a test that the old code was not covering?

Reply:
Actually there is a better way, 
`self.hour % 12  or 12`
I'll use it in my next commit.